packages/cashier-v2/src/types.ts

```diff
@@ -34,6 +34,7 @@ declare module 'react-router-dom' {
             state: Record<string, unknown>;
         };
         push: (path: TRouteTypes.TRoutes) => void;
+        goBack: () => void;
```

Comment:
why do we need this change in cashier-v2? :)

Reply:
@ahmadtaimoor-deriv ?

Reply:
I am using useHistory goBack in header and I was getting type error it's getting the type from so I had to add it here